addons/crm/models/crm_lead.py

```diff
@@ -10,6 +10,7 @@
 from odoo.osv import expression
 from odoo.tools.translate import _
 from odoo.tools import email_re, email_split
+from odoo.tools.date_utils import add
```

Comment:
Is it necessary to use add although a simple standard timedelta / relativedelta could do it ?